### PR TITLE
Add velocities to netcdf

### PR DESCRIPTION
**Pull request by jlmaurer**
This is a simple start to adding velocities to the output NETCDF file. This allows the interpolated velocities to be viewed in addition to the strain rates. 
- Velocities are not currently output by all of the methods, so for now I'm just having nans get put in place. This can be addressed later, as I'm pretty sure almost all of the methods have ways of getting velocity too, it's just not currently being done
- I also simplified one area by scaling velocities instead of strains

### CODE REVIEW

**Comment by jlmaurer** [comment]
@kmaterna this PR comes next in my list, adding in (mostly placeholders for) velocities to the output NETCDF file. I think most methods have the capability to calculate velocities (I know gpsgridder and visr do, and I'm pretty sure huang can as well). Obviously the delaunay methods don't, or at least I couldn't think how they could. #8 depends on this one, so I'm leaving that one as a draft for now.

File: Strain_Tools/strain/models/strain_visr.py [hunk 1]
```diff
@@ -60,7 +60,11 @@ def compute_visr(myVelfield, strain_range, inc, distwgt, spatwgt, smoothincs, ex
     subprocess.call(['mv', strain_data_file, tempdir], shell=False);
     subprocess.call(['mv', strain_output_file, tempdir], shell=False);
     print("Success computing strain via Visr method.\n");
-    return [xdata, ydata, rot, exx, exy, eyy];
+
+    # I know visr allows for velocity calculation, but dropping this in here now as a placeholder
+    Ve, Vn = np.nan*np.empty((4,4)), np.nan*np.empty((4,4))
```
Comment (kmaterna) [inline Strain_Tools/strain/models/strain_visr.py:65]: why shape = (4,4) ?  This is breaking on my machine for visr and delaunay, which use this line

Comment (jlmaurer) [reply Strain_Tools/strain/models/strain_visr.py:65]: @kmaterna I think that is a typo. I may have fixed it on my other machine and not pushed the update. I'll push an update!

File: Strain_Tools/strain/models/strain_gpsgridder.py [hunk 2]
```diff
@@ -77,16 +77,7 @@ def compute_gpsgridder(myVelfield, range_strain, inc, poisson, fd, eigenvalue, t
...
-    # Lastly we multiply by 1000 for units
-    exx = exx.T;
-    eyy = eyy.T;
-    exy = exy.T;
-    rot = rot.T;
-    exx = np.multiply(exx, 1000);
-    exy = np.multiply(exy, 1000);
-    eyy = np.multiply(eyy, 1000);
-    rot = abs(np.multiply(rot, 1000));
+    [exx, eyy, exy, rot] = strain_tensor_toolbox.strain_on_regular_grid(xinc, yinc, udata/1000, vdata/1000)
```
Comment (kmaterna) [inline Strain_Tools/strain/models/strain_gpsgridder.py:80]: @jlmaurer just tested this - the line should be (..., udata*1000, vdata*1000). Then I get the same answers. could you add that and then I'll merge?

Comment (kmaterna) [reply Strain_Tools/strain/models/strain_gpsgridder.py:80]: I'm adding a few changes (pygmt long-form-arguments, etc.) to my branch but there will be no conflicts

Comment (jlmaurer) [reply Strain_Tools/strain/models/strain_gpsgridder.py]: @kmaterna done! Also fixed a bug I had in the geostats module with the dx calcualtion

**Comment by fengnian-chang** [comment]
This is indeed an important update, as I have the possibility to combine the GNSS interpolated Vn and the InSAR Ve to calculate the strain rate. I'm successfully extracting velocities from '***_strain.nc' in geostats, gpsgridder and loc_avg_grad methods, but I get an error for delaunay. Is there any way to fix it? 
error

Thanks for your great software!

**Pull request merged**